src/File.php

```diff
@@ -70,6 +71,7 @@ class File implements Response
     public $redirects = 0;
     public $error;
     public $method = \SimplePie\SimplePie::FILE_SOURCE_NONE;
+    /** @var string Holds the original requested URL */
```

Comment:
That is not true. If a permanent redirect is encountered, it will hold the redirected URL:

https://github.com/simplepie/simplepie/blob/e597f27d414a774f732c0b260d67e11f6b6a7524/src/File.php#L139

Reply:
Yes, that is the only exception. But we can also remove the comments. I only added them while understanding the meanings of them.

Reply:
I think the comment is useful. The meaning of the property is not obvious and I have to check the code every time to make sense of it.

Reply:
I fixed the comment.